timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -0,0 +1,156 @@
+                ><i class="fas fa-copy"></i></span>
+        </b-table-column>
+        <b-table-column field="es_query" label="ES Query" v-slot="props" sortable searchable>
+          {{ props.row.es_query }}
+                <span class="icon is-small">
+                <router-link :to="{ name: 'Explore', query: {q : props.row.es_query} }"
+                   ><i class="fas fa-search"></i></router-link></span>
+                <span class="icon is-small"
+                style="cursor:pointer;float:right;"
+                title="Copy value"
+                v-on:click="copyCode(props.row.es_query)"
+                ><i class="fas fa-copy"></i></span>
+        </b-table-column>
+        <b-table-column field="file_name" label="File Name" v-slot="props" sortable searchable>
+          {{ props.row.file_name }} <span 
+                class="icon is-small" 
+                @mouseover="hover = true"
+                @mouseleave="hover = false"
+                :class="{ active: hover }"
+                style="cursor:pointer;float:right;"
+                title="Copy value"
+                v-on:click="copyCode(props.row.file_name)"
+                ><i class="fas fa-copy"></i></span>
+                <span class="icon is-small">
+                <router-link
+                      :to="{ name: 'Explore', query: {q : props.row.file_name} }"
+                   ><i class="fas fa-search"></i></router-link></span>
+        </b-table-column>
+        <b-table-column field="tags" label="Tag" v-slot="props" sortable searchable>
+          <ul class="content-list">
+            <li style="padding:10px;border-bottom:none;cursor:pointer;">
+              <div v-for="tag in props.row.tags" v-bind:key="tag">
+                {{ tag }} 
+                <span class="icon is-small">
+                <router-link
+                      :to="{ name: 'Explore', query: {q : tag} }"
+                   ><i class="fas fa-search"></i></router-link></span>
+            </div>
+            </li>
+          </ul>
+        </b-table-column>
+      </b-table>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      sigma_list: [],      
+      currentPage: 1,
+      ascending: false,
+      sortColumn: '', 
+      perPage: 10,
+      upHere : false,
+      hover : false     
+    }
+  },
+  methods: {
+    copyCode: function (value) {
+          const el = document.createElement('textarea');
+          el.value = value;
+          document.body.appendChild(el);
+          el.select();
+          document.execCommand('copy');
+          this.$buefy.notification.open('Copied')
+          document.body.removeChild(el);
+    }
+  },
+  computed: {
+    sigmas() {
+      return this.sigma_list
+    },
+    sketch() {
+      return this.$store.state.sketch
+    },
+    meta() {
+      return this.$store.state.meta
+    },
+  },
+  created() {
+    this.sigma_list =  this.$store.state.sigma_list
```

Comment:
This is redundant with the computed property `sigmas` above.
Just change the computed to do somehing like this:

```
sigmaRules() {
  return this.$store.state.sigma_list
}
```

Then you don't need to do this in the created() lifecycle hook.
See here for mote details on computed properties: https://vuejs.org/v2/guide/computed.html